misc/probe-client/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>keycloak-misc-parent</artifactId>
+        <groupId>org.keycloak</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <groupId>org.keycloak</groupId>
+    <artifactId>keycloak-probe-client</artifactId>
+    <name>Probe Client</name>
+    <description>A very basic HTTP CLI to replace curl in container live/ready probes</description>
+    <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
```

Comment:
Do we really need http client or we can just use plain Java? That way we don't need to create an uber-jar.

Reply:
+1 but if possible, I'd like to avoid having the probe-client and rather rely on the native httpGet probe as part of K8s.

Reply:
Right, Java 11 has a built-in client now. I didn't see it being used anywhere in the kc source code, so I stuck with what was in use already. Could be easily changed

Reply:
Will be going with the built-in k8s probes. So this will be deleted when I next push